app/lib/actions/katello/host/erratum/applicable_errata_install.rb

```diff
@@ -12,7 +12,9 @@ def plan(host, options = {})
 
             applicable_errata = host.content_facet.applicable_errata
             applicable_errata = applicable_errata.with_identifiers(errata_ids) unless options[:update_all]
-            plan_action(Actions::Katello::Host::Erratum::Install, host, applicable_errata.pluck(:errata_id))
+            unless applicable_errata.empty?
+              plan_action(Actions::Katello::Host::Erratum::Install, host, applicable_errata.pluck(:errata_id))
```

Comment:
Any opinion on providing a message to the task output here (I think it's possible) to the effect of "No applicable errata, skipping". I'm imagining a user or support comparing the task details between the two variants and wondering about the differences in the sub-plans

Reply:
How about a task output like so? (As seen in the raw task output for the ApplicableErrataInstall for a host that is skipped.)
![Screenshot from 2020-03-25 08-21-32](https://user-images.githubusercontent.com/266755/77535937-db6dfe00-6e71-11ea-9e7b-a67e52477157.png)

Reply:
Perfect!